### PR TITLE
Added Al Sharq TV. Fixed inconsistent hyphenation of Arabic Al-prefix.

### PR DESCRIPTION
**Pull request by 0x41mmarVM**
Al Sharq TV: Stream tested and is reliable. Works in both Kodi and VLC, for hours.

Al-prefix:
AL- is the quivalent of "The" in Arabic, and can either be spelled "Al-Something" or "Al Something". All Arabic channel names in the repo were using the space spelling except for the 4 fixed in this PR. The inconsistency caused an issue with alphabetical sorting where hyphenated names fall to the bottom.

### CODE REVIEW

File: channels/my.m3u [hunk 1]
```diff
@@ -29,6 +29,8 @@ http://210.210.155.35/uq2663/h/h07/01.m3u8
 http://stream2.ninmedia.tv/nabawitv/nabawitv_360p/playlist.m3u8
 #EXTINF:-1 tvg-id="NTV7" tvg-name="NTV7" tvg-language="" tvg-logo="https://freeview.github.io/iptv/logos/ntv7.png" group-title="",NTV7
 https://raw.githubusercontent.com/exodiver/IPTV/master/M3U8/Token/NTV7.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://yt3.ggpht.com/a/AATXAJzd2RPKopBOqInaP7TGFkpOzRgbwaApn81UBA" group-title="Entertainment",Pertiwi TV
+https://mn-nl.mncdn.com/pertiwitv/pertiwitv/playlist.m3u8
```
Comment (freearhey) [inline channels/my.m3u:33]: The link is not working at the moment. Tested in streamtest.in, Elmedia Video Player and VLC player.
Captura de pantalla 2020-05-28 a las 12 25 52
Captura de pantalla 2020-05-28 a las 12 26 01
Captura de pantalla 2020-05-28 a las 12 26 23

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

File: channels/my.m3u (hunk 1 above)
Comment (0x41mmarVM) [reply channels/my.m3u:33]: That is strange, perhaps a temporary downtime? I've just checked it now in VLC, ffmpeg, and videoJS. Seems to be working ok.

Comment (freearhey) [reply channels/my.m3u:33]: You're probably right. The channel's working fine right now.

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
Thank you for your help!

**Pull request merged**